content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +93,186 @@ there are any errors on the work entry, such as required fields being left blank
-:guilabel:`Name` field, typically listing the month and year for the particular batch.
+A batch entry appears for the time period selected. The batch name populates the :guilabel:`Name`
+field in a default `From (date) to (date)` format.
 
-The date range the payslips apply to appears in the :guilabel:`Period` field. The company appears in
-the :guilabel:`Company` field, along with an option to mark the payslips as a credit note. To make
-changes, click the :guilabel:`Edit` button in the top left, make any changes, then click either
-:guilabel:`Save` to accept the changes or :guilabel:`Discard` to revert to the original data.
-
-.. image:: work_entries/batch.png
-   :align: center
-   :alt: Batch information that appears when making a batch.
+The date range the payslips apply to appears in the :guilabel:`Period` field, and the company
+appears in the :guilabel:`Company` field. It is not possible to make changes to this form.
 
 Click the :guilabel:`Create Draft Entry` button to create the payslips for the batch.
 
-Click on the :guilabel:`Payslips` button in the top right to view all the payslips for the batch.
+Click the :guilabel:`Payslips` smart button in the top right to view all the payslips for the batch.
+
+.. image:: work_entries/generate-payslips.png
+   :align: center
+   :alt: Information that appears when generating payslips.
 
 .. note::
-   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button has been
+   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button is
    clicked. After, the payslip status will change to *Done*.
 
-Payslips can be printed by clicking the box next to each payslip to print, or clicking the box next
-to :guilabel:`Reference` to select all the payslips at once. Click the :guilabel:`Print` button, and
-a PDF file will be created with all the specified payslips.
+Printing payslips
+-----------------
+
+To print payslips, first view the individual payslips by clicking the :guilabel:`Payslips` smart
+button on the batch form. Next, select the payslips to print in the payslip list. Click the box next
+to each payslip to print, or click the box next to :guilabel:`Reference` to select all the payslips
+in the list at once. Click the :guilabel:`Print` button, and a PDF file is created with all the
+specified payslips.
 
 .. image:: work_entries/print-payslips.png
    :align: center
    :alt: Print button for printing the payslips.
 
+.. note::
+   The :guilabel:`Print` button does not appear until at least one payslip is selected in the list.
+
+Time Off to Report
+==================
+
+If a time-off request is submitted for a time period that was already processed on a payslip, the
+time-off request appears in the :guilabel:`Time Off to Report`, with a status of :guilabel:`To defer
+to next payslip`. This is because the employee was already paid for that day, and it was logged as
+time spent at work, as a typical work day.
```

Comment:
```suggestion
If a time off request is submitted for a time period that was already processed on a payslip, the
time off request appears in the :guilabel:`Time Off to Report`, with a status of :guilabel:`To defer
to next payslip`. This is because the employee was already paid for that day, and it was logged as
time spent at work, as a typical work day.
```